test/Graphics/Drawable.test.cpp

```diff
@@ -1,4 +1,7 @@
 #include <SFML/Graphics/Drawable.hpp>
+#include <SFML/Graphics/RenderTexture.hpp>
```

Comment:
Luckily `Drawable` comes before `RenderTarget` alphabetically but if the opposite was true then the header under test would not be compiled first meaning we wouldn't be able to detect missing `#include`s within Drawable.hpp. I hope we can find some way to enforce that the header under test always gets sorted first by clang-format.

Reply:
If you add a comment in the lines in between the `#include`s, will clang-format still mess with the order?

Reply:
believe adding a comment will suffice but I haven't tested that yet